tests/unittests/tests-core/tests-core-ringbuffer.c

```diff
@@ -33,18 +33,18 @@ static ringbuffer_t rb = RINGBUFFER_INIT(rb_buf);
 static mutex_t mutex;
 static kernel_pid_t pid_add, pid_get;
 
-static void assert_avail(unsigned assumed)
+static void expect_avail(unsigned assumed)
```

Comment:
Same in this file.

Reply:
(IMHO the change on this whole file should be reverted)

Reply:
I have it dropped the commit already.